manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: UnigramDev.Unigram
+PackageVersion: 8.9.7682.0
+PackageLocale: en-US
+Publisher: UnigramDev

```

```suggestion
Publisher: Telegram FZ-LLC
```
This must match Control Panel.